templates/packer/ansible/roles/common/files/worker-initialization-scripts/set_executor_hostname.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
(slightly unrelated) Fwiw, `pipefail` has it's own set of weirdness , see https://mvogt.wordpress.com/2018/07/23/bash-o-pipefail-can-have-pitfailswpitfalls/ - not that it matters here.

Reply:
oh interesting, had no idea